sql/src/test/java/io/crate/planner/operators/JoinTest.java

```diff
@@ -180,4 +180,28 @@ public void testMixedHashJoinNestedLoop() {
         assertThat(join.left(), instanceOf(Join.class));
         assertThat(((Join)join.left()).joinPhase(), instanceOf(HashJoinPhase.class));
     }
+
+
+    @Test
+    public void testOrderByIsPushedDownToLeftSide() {
```

Comment:
We can move this to the `PushDownTest` class I think, or please check if the case is already covered there.